### ISSUE TITLE
Content for Projects Page

### ISSUE THREAD

**Issue by nohorse** [EugTech/OpenEugeneFestivalWeb#18]
up near the top is a call to action

Submit a project
We are looking for projects that collaborators can work on during the event. Maybe you have a project that is up and running but not open-sourced. We can help take you to the next level. Perhaps you have an idea for a project and would like to start it out by working in the open. We can get you started. Or maybe you have some data stuck in a flat file somewhere you’d like to see made available to the world or have a PI built. That works too! We’d love to build community connections between projects and contributors.

Fill in the project form here  https://goo.gl/forms/xQsoEPznuxRDdhNz2

then list all the cool projects (looks great)

maybe ask for link to projects or images?    cc: admin@eugenetech.org on emails.

**Comment by mckelveygreg** [comment]
added description and link to form

**Comment by nohorse** [comment]
I just updated it with 


Current State of your project | Possible state after the event
-- | --
I have a great idea! | You have launched a project online with all the things folks need to   collaborate with you!
I have an open project on github but nobody is contributing. | Now your project is welcoming and inviting to folks and you have   issues for folks to work on.
I have a project I’d like to share, but I’d like to keep my branding private   for reasons. | Now the guts of your project is open, but your branding is protected   by a licenses that preserves control over your brand.
I made a giant pong controller and now folks in Chattanooga want one,   but I don’t want to build it or ship it to them | You’ve publish the plan on Instructables, add a creative commons   license and they can build it themselves!
